tests/unit/lang/translator/de.txt

```diff
@@ -0,0 +1,162 @@
+\nBootloader Version\n%d.%d.%d\n\nBuddy Board\n%d.%d.%d\n%s
+Ein Crash-Dump-Bericht (Datei dump.bin) wurde auf dem USB-Laufwerk gespeichert.
+Alle Tests erfolgreich abgeschlossen!
+Immer
+Anwenden
+Druck wirklich beenden?
+Assist
+Auto Home
+AutTn
+ZURÜCK
+BlindAlert
+KnopfEcho
+Kalibrierung
+Kalibrierung erfolgreich!\nFrohes Drucken!
+FILAMENT WECHSEL
+Filament Wechsel
+Stahlblech reinigen.
+Glückwunsch! Die XYZ-Kalibrierung ist in Ordnung. XY-Achsen sind senkrecht.
+WEITER
+Abkühlen
+SENSOR AUSSCHALTEN
+Motoren ausschalten
+Möchten Sie den\nletzten Schritt\nwiederholen und\nden Abstand zwischen\nDüse und Heizbett\nneu einstellen? 
```

Comment:
What is the character per line limitation?

Reply:
It depends on the GUI component size (rectangle). There are still issues with line breaks, therefore there are those hard \n